docs/BUILD.bazel

```diff
@@ -3,46 +3,97 @@ load("@aspect_bazel_lib//lib:docs.bzl", "stardoc_with_diff_test", "update_docs")
 stardoc_with_diff_test(
     name = "js_binary",
     bzl_library_target = "//js/private:js_binary",
+    # https://github.com/bazelbuild/stardoc/pull/141
+    target_compatible_with = select({
+        "//:bzlmod": ["@platforms//:incompatible"],
+        "//conditions:default": [],
+    }),
```

Comment:
Stardoc doesn't work with bzlmod enabled.

Reply:
fix is coming soon I hear. this is an interesting workaround